src/components/ReviewPage.jsx

```diff
@@ -24,8 +25,9 @@ function ReviewPage() {
             setReview(reviewFromAPI[0])
             setReviewLoading(false)
             setVoteCount(reviewFromAPI[0].votes)
+            setCommentCount(reviewFromAPI[0].comment_count)
         })
-    }, [review_id])
+    }, [review_id, commentCount])
```

Comment:
`commentCount` shouldn't be a dependency here as it will infinite loop when you use `setCommentCount`